src/settings/WallabagSettingTab.ts

```diff
@@ -47,17 +47,7 @@ export class WallabagSettingTab extends PluginSettingTab {
         name: 'Article note template file',
         desc: sanitizeHTMLToDom(
           'The template file that will be used for the new articles.<br> ' +
-          '<code>{{article_title}}</code>: Title of the article.<br>' +
-          '<code>{{original_link}}</code>: Link to the original article.<br>' +
-          '<code>{{created_at}}</code>: Creation date of the article in Wallabag.<br>' +
-          '<code>{{wallabag_link}}</code>: Link to the Wallabag article.<br>' +
-          '<code>{{content}}</code>: HTML Content extracted by Wallabag. <br>' +
-          '<code>{{pdf_link}}</code>: An Obsidian link to the exported pdf.' +
-          '<code>{{tags}}</code>: Tags of the article.<br>' +
-          '<code>{{reading_time}}</code>: Reading time of the article.<br>' +
-          '<code>{{preview_picture}}</code>: Preview picture of the article.<br>' +
-          '<code>{{domain_name}}</code>: Domain name of the article.<br>' +
-          '<code>{{annotations}}</code>: Created annotations of the article.<br>'
+          'See <a href="https://github.com/huseyz/obsidian-wallabag">documentation</a> for examples.'
```

Comment:
At this point there are so many template fields, I thought it'd be better to just directly link to the docs for a full list.